.github/workflows/build.yml

```diff
@@ -11,6 +11,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Install system dependencies
+        run: | 
+         sudo apt-get update  -y 
+         sudo apt-get install -y gcc-12 g++-12 libnuma-dev
```

Comment:
also it seems we have split the vLLM installation in 2 parts with gcc being installed here and rest of it getting installed in tox.

Any reason for this split ?